Redis

HyunsungJo opened the pull request:
add get vertices

wishoping commented on s2core/src/main/scala/com/kakao/s2graph/core/storage/redis/RedisRPC.scala:
```diff
@@ -59,7 +63,12 @@ case class RedisGetRequest(key: Array[Byte], isIncludeDegree: Boolean = true) ex
 
-case class RedisPutRequest(key: Array[Byte], qualifier: Array[Byte], value: Array[Byte], timestamp: Long) extends RedisRPC(key)
+case class RedisPutRequest(key: Array[Byte], qualifier: Array[Byte], value: Array[Byte], timestamp: Long) extends RedisRPC(key) {
+  val k = GraphUtil.bytesToHexString(key)
```
`lazy val` looks better than `val`

wishoping commented on s2core/src/main/scala/com/kakao/s2graph/core/storage/redis/RedisRPC.scala:
```diff
@@ -68,5 +77,9 @@ case class RedisAtomicIncrementRequest(key: Array[Byte], qualifier: Array[Byte]
 
-case class RedisDeleteRequest(key: Array[Byte], value: Array[Byte], timestamp: Long) extends RedisRPC(key)
+case class RedisDeleteRequest(key: Array[Byte], value: Array[Byte], timestamp: Long) extends RedisRPC(key) {
+  val k = GraphUtil.bytesToHexString(key)
```
`lazy val` looks better than `val`

wishoping commented on s2core/src/main/scala/com/kakao/s2graph/core/storage/redis/RedisRPC.scala:
```diff
@@ -20,6 +21,9 @@ case class RedisGetRequest(key: Array[Byte], isIncludeDegree: Boolean = true) ex
   lazy val degreeEdgeKey = Bytes.add(key, RedisRPC.DEGREE_EDGE_POSTFIX_BYTE)
 
+  // Todo: Not sure if limiting the number of properties is the right approach..
+  lazy val MaxPropNum = 100
```
Why do you use limit number for vertex properties? If `zrangebylex` supports unlimit count as a parameter, we'd better use the unlimit count. Do you consider about fetch size because of rpc timeout?

HyunsungJo commented:
@wishoping 
Forgot that offset + count were optional for zrangebylex. Fixed not to use offset + count for vertex get.
Changed val to lazy val, as well.
